.github/workflows/rebase.yaml

```diff
@@ -0,0 +1,37 @@
+name: Automated rebase
+
+on:
+  issue_comment:
+    types:
+      - created
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  rebase:
+    name: Rebase
+
+    runs-on: ubuntu-24.04
+    environment: automated
+
+    if: >-
+      github.event.issue.pull_request != '' && 
+      (
+        contains(github.event.comment.body, '/rebase') || 
+        contains(github.event.comment.body, '/autosquash')
+      )
```

Comment:
github provides `rebase` and squash options when merging

Reply:
Already removed